projects/configs/ms-pinpoint.yaml

```diff
@@ -18,6 +18,7 @@ project:
 
 tag_operation_files:
     base:
+        - "base.yaml"
```

Comment:
I think you'll also need to add "base.yml" into the other project configs in this directory for tests to pass

Reply:
[adds base.yml to project files but ./test-extract-uclh-omop-cdm.yaml …failed with #- `base.yaml` with 2 tests failed](https://github.com/SAFEHR-data/PIXL/pull/498/commits/0505a9aa730d586dd653b6e6556271211d7d483d) 
[0505a9a](https://github.com/SAFEHR-data/PIXL/pull/498/commits/0505a9aa730d586dd653b6e6556271211d7d483d)